go.mod

```diff
@@ -1,8 +1,5 @@
 module github.com/hashicorp/packer-plugin-sdk
 
-// As described in https://github.com/hashicorp/packer-plugin-sdk/issues/187
-replace github.com/zclconf/go-cty => github.com/nywilken/go-cty v1.12.1
```

Comment:
Seeing as we are validating the replace during testing to check for potential issue what do you think about not bumping github.com/zclconf/go-cty and just keeping it pinned to 1.10.0? 

Will that cause issues?

Reply:
This def limits the version of Hcl/v2 to as hcl/v2@2.15.0 requires at least 1.12.0 of zclconf/go-cty

Reply:
Issues no, but that will not make the compile-time error appear, so users won't be hinted at running `packer-sdc fix .` when they upgrade

Reply:
I'm talking about just in the SDK. The compile time check will fail if a user tries to include the SDK with an unsupported version of go-cty. So they should still see the hint.